pkg/readers/vsix_ext_reader.go

```diff
@@ -12,6 +12,12 @@ import (
 
 const (
 	vsCodeExtensionExtensionsFileName = "extensions.json"
+
+	// Default paths for IDE extension directories relative to home directory
+	vsCodeExtPath   = ".vscode/extensions"
```

Comment:
Should we maintain this as a map of editor name and a map of path and ecosystem?